### PR TITLE
fix missing-prototypes warnings in torch_cpu (Part 1)

### PR DESCRIPTION
**Pull request by cyyever**
This PR fixes some missing-prototypes violations in the torch_cpu source as planed in PR #96805. The final object is to add the Wmissing-prototypes option to torch_cpu building.

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @mcarilli @ptrblck @leslie-fang-intel

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Thanks for taking the time to do this. Our c++ code health is really not that great :( 

I assume there will be more parts and the final one will add the proper build flags?

**Comment by cyyever** [comment]
@albanD We can gradually improve its quality and the use of static functions provides more opportunity for compiler optimizations. After fix all violations, I would add a build flag.

File: aten/src/ATen/native/verbose_wrapper.cpp [hunk 1]
```diff
@@ -25,7 +25,7 @@ TORCH_API int _mkl_set_verbose(int enable) {
 #endif
 }
 
-TORCH_API int _mkldnn_set_verbose(int level) {
```
Comment (malfet) [inline aten/src/ATen/native/verbose_wrapper.cpp:28]: 👍

**Pull request closed without merging**